frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1458,6 +1495,49 @@ const RulesPage_: React.FC<Rules> = ({ data, loaded, loadError }) => {
     },
   ];
 
+  const rulesRowFilters: RowFilter[] = [
+    alertStateFilter(),
+    severityRowFilter(),
+    {
+      defaultSelected: [AlertSource.Platform],
+      filterGroupName: t('public~Source'),
+      items: [
+        { id: AlertSource.Platform, title: t('public~Platform') },
+        { id: AlertSource.User, title: t('public~User') },
+      ],
+      reducer: alertingRuleSource,
+      type: 'alerting-rule-source',
+    },
+  ];
+
+  const RuleTableRow: RowFunction<Rule> = ({ index, key, obj, style }) => (
```

Comment:
Let's move `RuleTableRow` back outside this component to keep `RulesPage` smaller. `RuleTableRow` will just need to call `useTranslation` itself.